app/views/components/footer/shared.html.haml

```diff
@@ -56,8 +56,10 @@
         %hr.c-divider.--small
         %ul
           %li= link_to "Exercism's Docs", docs_path
+          / # TODO
```

Comment:
@ErikSchierboom Could you do this one pls. Path would be `using/getting-started`. I'm sure we have some old copy for this somewhere? in website-copy maybe.

Reply:
Done

Reply:
@ErikSchierboom Feel free to push to this PR. This would be `doc_path(:using, "getting-started")` if you're not familiar with the syntax.